src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.ts

```diff
@@ -34,53 +34,48 @@ export class GeneSoeChartsComponent implements OnInit {
   }
 
   @Input() wikiId = '';
-  charts: any[] = [];
 
   barPrimaryColor = '#8b8ad1';
   barAlternativeColor = '#42C7BB';
 
+  scoreDistributions: OverallScoresDistribution[] = [];
+
   constructor(private geneService: GeneService) {}
 
   ngOnInit() {}
 
-  init() {
-    this.geneService.getDistribution().subscribe((data: any) => {
-      let overallScoreDistribution = data.overall_scores;
-
-      overallScoreDistribution.sort((a: any, b: any) =>
-        a.name > b.name ? 1 : -1
-      );
+  sortScoreDistributions(distributions: OverallScoresDistribution[]) {
+    distributions.sort((a: OverallScoresDistribution, b: OverallScoresDistribution) => {
+      if (a.name === 'Target Risk Score') {
```

Comment:
Consider adding a comment that explains this sort, because it's not really clear to me what the goal is.

Reply:
I'll rename the method and add a comment.